### ISSUE TITLE
Stopped working

### ISSUE THREAD

**Issue by abdulhkeemGit** [endeavoursc/firefox-overlay-scrollbars-win10#6]
firefox 72 broke this awesome scrollbars can you fix them please, thanx.

**Comment by zakutin** [comment]
Will there be any updates?

**Comment by abdulhkeemGit** [comment]
found a fix go to %appdata%\local 
delete the "Mozilla" folder now they work again!

**Comment by zakutin** [comment]
Hm.. Thank you, I was aware of this fix, but unfortunately it didn't work for me.
Even if the option `toolkit.legacyUserProfileCustomizations.stylesheets` enabled or disabled.
Could you please check, if it is on/off in your `about:config`?

**Comment by abdulhkeemGit** [comment]
set to true, sorry for late reply.

**Comment by Deadsy81** [comment]
None of that does anything for me.

**Comment by abdulhkeemGit** [comment]
that's all i did before it worked again for me.
try adding these lines to "usercontent.css":
* {
	scrollbar-color: #444 #1f1f1f !important;
	scrollbar-width: thin !important;
}

**Comment by zakutin** [comment]
But there is no `usercontent.css` in git repository. Shall we create it?

**Comment by Deadsy81** [comment]
> 
> 
> that's all i did before it worked again for me.
> try adding these lines to "usercontent.css":
> 
>     * {
>       scrollbar-color: #444 #1f1f1f !important;
>       scrollbar-width: thin !important;
>       }

Yeah, I know about that, but it's not as good as the original.

**Comment by Deadsy81** [comment]
> 
> 
> But there is no `usercontent.css` in git repository. Shall we create it?

Yes

**Comment by abdulhkeemGit** [comment]
yes create it then delete the "mozilla" folder in appdata\local see what happens.

**Comment by zakutin** [comment]
@abdulhkeemGit , could you please upload all the files related to your working configuration in your chrome folder?

**Comment by abdulhkeemGit** [comment]
sure; but i have shadowfox mixed in with the lines too.
[Chrome.zip](https://github.com/endeavoursc/firefox-overlay-scrollbars-win10/files/4049284/Chrome.zip)

**Comment by LoLei** [comment]
I've had this happen multiple times, after Firefox updates the overlay scrollbar would be gone. Re-downloading and pasting the files worked sometimes, other times it took a reboot. This time however it stayed broken for multiple days, and still remains so.

**Comment by rafaelmardojai** [comment]
XBL was removed, therefore there is no more `-moz-binding`.
See: https://developer.mozilla.org/en-US/docs/Archive/Mozilla/XBL

**Comment by abdulhkeemGit** [comment]
userChrome.xml was in XBL converted it, hope it won't stop again, thanks.

**Comment by endeavoursc** [comment]
For now the best solution is using this:

1) download _userChrome.js_ from here and rename it to _scrollbars.uc.js_
2) download _install_folder_ with its contents and _userChrome.js_ from https://github.com/alice0775/userChrome.js/tree/master/72
3) put _userChrome.js_ and _scrollbars.uc.js_ in the _profile/chrome_ folder
4) put the _install_folder_ **contents** in the firefox install folder (ex. _program files/mozilla firefox_)
5) clear cache (delete the profile from _%localappdata%\Mozilla\Firefox\Profiles_)

and it should work again.

**Comment by LoLei** [comment]
@endeavoursc It worked thanks.

On Linux:
* Profile dir can be found in Firefox with Help>Troubleshooting Information
* Installation folder is `/usr/lib/firefox` (At least on Arch Linux)
* Cache clearing was not necessary.

**Comment by webdevs-pro** [comment]
Work for me now!

**Comment by Movisivi** [comment]
@LoLei 

Just wanted to say thank you! I was getting this nasty blue colored scrollbar both vertically and horizontally. This still works as of Windows 10 v2004 and latest Firefox Nightly.

**Comment by hbRRR** [comment]
Doesn't (stopped) work in Firefox 92.0 (64-bit).
Any solution?
